include/game.h

```diff
@@ -288,40 +288,43 @@ typedef enum {
     FLAG_DESTROY_IF_OUT_OF_CAMERA = 0x80000000,
 } EntityFlag;
 
-// document g_Player.unk0C
-#define PLAYER_STATUS_BAT_FORM 0x1
-#define PLAYER_STATUS_MIST_FORM 0x2
-#define PLAYER_STATUS_WOLF_FORM 0x4
-#define PLAYER_STATUS_TRANSFORM                                                \
-    (PLAYER_STATUS_BAT_FORM | PLAYER_STATUS_MIST_FORM | PLAYER_STATUS_WOLF_FORM)
-#define PLAYER_STATUS_UNK8 0x8
-#define PLAYER_STATUS_UNK10 0x10
-#define PLAYER_STATUS_UNK_20 0x20
-#define PLAYER_STATUS_UNK40 0x40
-#define PLAYER_STATUS_STONE 0x80
-#define PLAYER_STATUS_UNK100 0x100
-#define PLAYER_STATUS_UNK200 0x200
-#define PLAYER_STATUS_UNK400 0x400
-#define PLAYER_STATUS_UNK800 0x800
-#define PLAYER_STATUS_UNK1000 0x1000
-#define PLAYER_STATUS_UNK2000 0x2000
-#define PLAYER_STATUS_POISON 0x4000
-#define PLAYER_STATUS_CURSE 0x8000
-#define PLAYER_STATUS_UNK10000 0x10000 // possibly freezing?
-#define PLAYER_STATUS_UNK20000 0x20000
-#define PLAYER_STATUS_UNK40000 0x40000
-#define PLAYER_STATUS_UNK80000 0x80000
-#define PLAYER_STATUS_UNK100000 0x100000
-#define PLAYER_STATUS_UNK200000 0x200000
-#define PLAYER_STATUS_UNK400000 0x400000
-#define PLAYER_STATUS_UNK800000 0x800000
-#define PLAYER_STATUS_AXEARMOR 0x1000000
-#define PLAYER_STATUS_ABSORB_BLOOD 0x2000000
-#define PLAYER_STATUS_UNK4000000 0x4000000
-#define NO_AFTERIMAGE 0x8000000
-#define PLAYER_STATUS_UNK10000000 0x10000000
-#define PLAYER_STATUS_UNK40000000 0x40000000
-#define PLAYER_STATUS_UNK80000000 0x80000000
+// document g_Player.status
+typedef enum {
+    PLAYER_STATUS_BAT_FORM = 0x1,
+    PLAYER_STATUS_MIST_FORM = 0x2,
+    PLAYER_STATUS_WOLF_FORM = 0x4,
+    PLAYER_STATUS_TRANSFORM =
+        (PLAYER_STATUS_BAT_FORM | PLAYER_STATUS_MIST_FORM |
+         PLAYER_STATUS_WOLF_FORM),
+    PLAYER_STATUS_UNK8 = 0x8,
+    PLAYER_STATUS_UNK10 = 0x10,
+    PLAYER_STATUS_UNK_20 = 0x20,
+    PLAYER_STATUS_UNK40 = 0x40,
+    PLAYER_STATUS_STONE = 0x80,
+    PLAYER_STATUS_UNK100 = 0x100,
+    PLAYER_STATUS_UNK200 = 0x200,
+    PLAYER_STATUS_UNK400 = 0x400,
+    PLAYER_STATUS_UNK800 = 0x800,
+    PLAYER_STATUS_UNK1000 = 0x1000,
+    PLAYER_STATUS_UNK2000 = 0x2000,
+    PLAYER_STATUS_POISON = 0x4000,
+    PLAYER_STATUS_CURSE = 0x8000,
+    PLAYER_STATUS_UNK10000 = 0x10000, // possibly freezing?
+    PLAYER_STATUS_UNK20000 = 0x20000,
+    PLAYER_STATUS_UNK40000 = 0x40000,
+    PLAYER_STATUS_UNK80000 = 0x80000,
+    PLAYER_STATUS_UNK100000 = 0x100000,
+    PLAYER_STATUS_UNK200000 = 0x200000,
+    PLAYER_STATUS_UNK400000 = 0x400000,
+    PLAYER_STATUS_UNK800000 = 0x800000,
+    PLAYER_STATUS_AXEARMOR = 0x1000000,
+    PLAYER_STATUS_ABSORB_BLOOD = 0x2000000,
+    PLAYER_STATUS_UNK4000000 = 0x4000000,
+    NO_AFTERIMAGE = 0x8000000,
+    PLAYER_STATUS_UNK10000000 = 0x10000000,
+    PLAYER_STATUS_UNK40000000 = 0x40000000,
+    PLAYER_STATUS_UNK80000000 = 0x80000000,
+} PlayerStateStatus;
```

Comment:
Not thrilled with this name but there is already an enumeration called `PlayerStatus` used for something else (related to familiars I think?)